tests/scattering1d/test_torch_backend_1d.py

```diff
@@ -173,19 +173,22 @@ def test_subsample_fourier(backend, device, random_state=42):
         assert "should be complex" in te.value.args[0]
 
 
-def test_unpad():
-    # test unpading of a random tensor
-    x = torch.randn(8, 4, 1)
+@pytest.mark.parametrize("backend", backends)
+@pytest.mark.parametrize("device", devices)
+def test_unpad(backend, device):
+    if backend.name == "torch_skcuda" and device == "cpu":
+        pytest.skip()
```

Comment:
Why are we adding this here? Can avoid these skips using the backend–device tuple scheme.

Reply:
could you explain this for me during the meeting?

Reply:
We'd have a list of compatible backends and devices and loop over those. So if skcuda is available, we'd have the list: `[(torch_backend, 'cpu'), (torch_backend, 'gpu'), (torch_skcuda_backend, 'gpu')` and parametrize the tests with that. This is implemented in the 2D Torch tests. No need to do it right now, but we should probably do this in the 1D and 3D tests as well.

Reply:
good idea